charts/spire/charts/spire-server/values.yaml

```diff
@@ -725,3 +725,24 @@ tests:
       pullPolicy: IfNotPresent
       version: ""
       tag: latest@sha256:60ccbc708bc17803b8253a1c4fdf61e9a68b55e6d0a8ae5735cc276ad4afa2d2
+
+## @section Namespaces
+namespaces:
+  ## @param namespaces.system.name Name of the Spire system Namespace.
+  ## @param namespaces.system.create Create a Namespace for Spire system resources.
+  ## @param namespaces.system.annotations [object] Annotations to apply to the Spire system Namespace.
+  ## @param namespaces.system.labels [object] Labels to apply to the Spire system Namespace.
+  system:
+    name: "spire-system"
+    create: false
+    annotations: {}
```

Comment:
I would set all the annotations from production example here by default. Especially because namespace creation is opt-in, that would only happen if the namespace creation is set to true.

Reply:
Unfortunately, the annotations aren't always consistent between configurations. They are a bit different in openshift and vanilla k8s and even between openshift deployments. So, I think we will need to make it programatic.

I do like the idea of making it easy to do the best practice annotations though...
I have another pr thats rolling in the best practices from the examples into the chart. maybe we add another one for the namespace annotations, like:
https://github.com/spiffe/helm-charts-hardened/pull/106/files#diff-e5d95a142cf1134e77e5c83dfeac07f49d1f433253a8b07d11a652c784403a6bR7-R9
?